src/components/PostList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Post from './Post';
+import {posts} from '../posts';
+
+class PostList extends React.Component {
```

Comment:
Use a function instead of class.
you didn't use the potentiality of class
https://ru.reactjs.org/docs/components-and-props.html#function-and-class-components